src/ads_globals.cpp

```diff
@@ -90,6 +90,17 @@ void hideEmptyParentSplitters(CDockSplitter* Splitter)
 	}
 }
 
+CDockInsertParam::CDockInsertParam(Qt::Orientation orient, bool bottomRight)
```

Comment:
What is the reason for providing all constructor for CDockInsertParam class?

Reply:
Do you mean the move constructor from QPair that is inherited by the **using QPair::QPair** declaration is causing compiler errors? I thought move constructors are supported by C++11? Which compiler is causing the error?

Reply:
Ubuntu gcc5 was complaining. the move constructor of QPair is explicitly declared and for some reason it wasn't feeding through correctly in the using directive

Reply:
Ok, this is not nice, that we need to change the code to fix a compiler problem but if this is the only solution for now, then it is o.k. for me.

Reply:
Forcing C++14 naturally forces this to work so I reverted my change